//Protocol buffer specification for storaged rpc server
syntax = "proto3";

package databaseRPC;

//this will generate the dbRPC class that contains a client stub
//and two interfaces for server to implement with corresponding methods
service dbRPC
{
  //send hello
  rpc Hello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest
{
  string name = 1;
  int32 message_id = 2;
}

message HelloReply
{
  string message = 1;
  int32 message_id = 2;
}
